### PR TITLE
chore: translations for login.properties

### PR DESCRIPTION
**Pull request by ui-okta**
## Description
Auto copy of latest translations by okta-i18n script.
### Bacon task details
- Task ID: de33aa5cbd1048c48fca274913d89bf5 
- Requested by: peng.yao@okta.com

## Notification sent to
@okta/eng-globalizationcore @haishengwu-okta @aarongranick-okta

### CODE REVIEW

File: packages/@okta/i18n/src/properties/login_de.properties [hunk 1]
```diff
@@ -109,9 +109,9 @@ errors.E0000030 = Ung\u00FCltige Anfrage. Ung\u00FCltiges Datum. Datumsangaben m
 errors.E0000031 = Ung\u00FCltige Suchkriterien.
 errors.E0000032 = Entsperren ist f\u00FCr diesen Benutzer nicht erlaubt.
 errors.E0000033 = Ung\u00FCltige Anfrage. In einer Anfrage kann nicht sowohl eine Suchabfrage als auch ein Filter angegeben werden.
-errors.E0000034 = "Passwort vergessen" ist f\u00FCr diesen Benutzer nicht erlaubt.
-errors.E0000035 = "Passwort \u00E4ndern" ist f\u00FCr diesen Benutzer nicht erlaubt.
-errors.E0000036 = "Sicherheitsfrage \u00E4ndern" ist f\u00FCr diesen Benutzer nicht erlaubt.
+errors.E0000034 = &quot;Passwort vergessen&quot; ist f\u00FCr diesen Benutzer nicht erlaubt.
+errors.E0000035 = &quot;Passwort \u00E4ndern&quot; ist f\u00FCr diesen Benutzer nicht erlaubt.
+errors.E0000036 = &quot;Sicherheitsfrage \u00E4ndern&quot; ist f\u00FCr diesen Benutzer nicht erlaubt.
```
Comment (mauriciocastillosilva-okta) [inline packages/@okta/i18n/src/properties/login_de.properties:114]: discussed over slack, these changes should be reverted.

File: packages/@okta/i18n/src/properties/login_ja.properties [hunk 2]
```diff
@@ -644,12 +645,13 @@ password.complexity.minAgeHours = \u6700\u5F8C\u306B\u30D1\u30B9\u30EF\u30FC\u30
...
 password.complexity.minAgeDays = \u6700\u5F8C\u306B\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u5909\u66F4\u3057\u3066\u304B\u3089\u5C11\u306A\u304F\u3068\u3082{0}\u65E5\u7D4C\u904E\u3057\u306A\u3051\u308C\u3070\u306A\u308A\u307E\u305B\u3093\u3002
 password.reset.verification = \u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u30EA\u30BB\u30C3\u30C8\u3059\u308B\u306B\u306F\u3001\u6B21\u306E\u3044\u305A\u308C\u304B\u306E\u8981\u7D20\u3067\u8A8D\u8A3C\u3057\u3066\u304F\u3060\u3055\u3044\u3002
+password.reset.revokeSessions = \u4ED6\u306E\u3059\u3079\u3066\u306E\u30C7\u30D0\u30A4\u30B9\u304B\u3089\u30B5\u30A4\u30F3\u30A2\u30A6\u30C8\u3057\u307E\u3059\u3002
 
 # The actual description consists of the password length message + one or more list elements
 # {0} is a number
 password.complexity.length = {0}\u6587\u5B57\u4EE5\u4E0A
 # {0} refers to a translated password requirement (denoted below), like "a lowercase letter" or "an uppercase letter"
-password.complexity.list.element = \u3001{0}
+password.complexity.list.element = , {0}
```
Comment (michaeltamaki-okta) [inline packages/@okta/i18n/src/properties/login_ja.properties:654]: Brought this up in a different PR, and Mau brought this up in Slack -- `,` should not be in the Japanese translation

File: packages/@okta/i18n/src/properties/login_zh_TW.properties [hunk 3]
```diff
@@ -192,7 +192,7 @@ errors.E0000110 = \u60A8\u5B58\u53D6\u7684\u9023\u7D50\u5DF2\u904E\u671F\u6216\u
 errors.E0000111 = {0} \u662F\u552F\u8B80\u7269\u4EF6\uFF0C\u56E0\u6B64\u7121\u6CD5\u4FEE\u6539\u3002
 errors.E0000112 = \u6B64\u4F7F\u7528\u8005\u4ECD\u8655\u65BC\u555F\u7528\u72C0\u614B\uFF0C\u56E0\u6B64\u7121\u6CD5\u66F4\u65B0\u3002\u8ACB\u5728\u5E7E\u5206\u9418\u5F8C\u518D\u8A66\u4E00\u6B21\u3002
 # {0} is a server-side error response, oftentimes coming directly from a vendor.
-errors.E0000113 = {0}\u3002
+errors.E0000113 = {0}.
```
Comment (michaeltamaki-okta) [inline packages/@okta/i18n/src/properties/login_zh_TW.properties:195]: Is this period `.` used in Chinese?

**Review by michaeltamaki-okta — APPROVED** [review]

File: packages/@okta/i18n/src/properties/login_zh_TW.properties (hunk 3 above)
Comment (mauriciocastillosilva-okta) [reply packages/@okta/i18n/src/properties/login_zh_TW.properties]: I reverted all the changes that replaced unicode with a dot or comma, we should be good now

**Review by nikhilvenkatraman-okta — APPROVED** [review]

**Pull request merged**